templates/add.html

```diff
@@ -0,0 +1,33 @@
+{% extends "layout.html" %}
+{% block title %}To-Do App{% endblock %}
+
+{% block navbar %}
+<div class="collapse navbar-collapse" id="navbarText">
+  <ul class="navbar-nav mr-auto">
+    <li class="nav-item">
+      <a class="nav-link" href="/">Home</a>
+    </li>
+    <li class="nav-item active">
+      <a class="nav-link" href="/add">Add</a>
+    </li>
+  </ul>
+</div>
+{% endblock %}
+
+
+{% block content %}
+  <div class="row">
+    <div class="col-md-12">
+      <form action="/add" method="post">
+        <h2>Add New Items</h2>
+        <div class="form-group">
+          <label for="todo">Description</label>
+          <input type="text" class="form-control" id="todo" name="new_todo">
```

Comment:
Probably worth adding a `required` attribute here for some server side validation.